src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java

```diff
@@ -191,11 +197,12 @@ public EveritWorklog(final ResultSet rs)
     MutableIssue issueObject = issueManager.getIssueObject(issueId);
     issue = issueObject.getKey();
     issueSummary = issueObject.getSummary();
-
+    if (StatusCategory.COMPLETE
```

Comment:
StatusCategory.COMPLETE is a String. And the check style don't like the == operator when comparing Strings.